### PR TITLE
test(ci) Test on Windows

### PR DESCRIPTION
**Pull request by Hywan**
Address #29 

Windows support is coming…

### CODE REVIEW

**Comment by Hywan** [comment]
Asking for help https://github.com/pyenv/pyenv/issues/1483.

**Comment by Hywan** [comment]
Changing the way we install Python on Github Actions, see #141. Hope it will solve the Windows issue.

File: src/memory/view.rs [hunk 1]
```diff
@@ -39,7 +39,7 @@ macro_rules! memory_view {
                 let view = self.memory.view::<$wasm_type>();
                 let offset = self.offset;
                 let range = if let Ok(slice) = index.cast_as::<PySlice>() {
-                    let slice = slice.indices(view.len() as i64)?;
+                    let slice = slice.indices(view.len() as _)?;
```
Comment (Hywan) [inline src/memory/view.rs]: Because sometimes it is `i32` depending of the CPU.

File: tests/test_module.py [hunk 2]
```diff
@@ -102,50 +102,50 @@ def test_exports():
         },
     ]
 
-def test_imports():
```
Comment (Hywan) [inline tests/test_module.py]: Because we have a bug in Wasmer. Will fix it. That's temporary.

Comment (syrusakbary) [reply tests/test_module.py]: Let's open an issue on this repo after merging the PR, so we don't forget :)

**Pull request merged**